modules/gapi/src/backends/render/grenderocv.cpp

```diff
@@ -19,18 +24,33 @@ GAPI_RENDER_OCV_KERNEL(RenderBGROCVImpl, cv::gapi::wip::draw::GRenderBGR)
             in.copyTo(out);
         }
 
-        cv::gapi::wip::draw::drawPrimitivesOCVBGR(out, prims, ftpr);
+        cv::gapi::wip::draw::drawPrimitivesOCVBGR(out, prims, state.ftpr);
+    }
+
+    static void setup(const cv::GMatDesc& /* in */,
+                      const cv::GArrayDesc& /* prims */,
+                      std::shared_ptr<RenderOCVState>& state,
+                      const cv::GCompileArgs& args)
+    {
+        using namespace cv::gapi::wip::draw;
+        auto has_freetype_font = cv::gapi::getCompileArg<freetype_font>(args);
```

Comment:
while this is valid in terms of `optional<>`, this may be hard to understand by a non-prepared person

Reply:
Fixed